solutions/swb-app/src/projectEnvironmentRoutes.ts

```diff
@@ -146,6 +146,8 @@ export function setUpProjectEnvRoutes(
       const envType = environment.ETC!.type;
       if (environment.status === 'STOPPING') {
         throw Boom.conflict('Cannot start environment while environment is currently being stopped');
+      } else if (['FAILED'].includes(environment.status)) {
```

Comment:
just use: environment.status === 'FAILED'

Reply:
can you double check if this error check should only happen at route level or service level. I'd lean towards the later

Reply:
Updated the conditional to be ` else if (environment.status === 'FAILED')` since we're only checking one string value.

Reply:
I think the error check on the route level makes sense because we'll want the same behavior for all environments. If we add the check at the service level, we will have to reuse the code for each new environment type. Also there's a precedence for having these types of checks on the route level. Examples [here](https://github.com/aws-solutions/research-service-workbench-on-aws/blob/release/4debe38e-5796-47b6-96ec-881a87e0df21/solutions/swb-app/src/projectEnvironmentRoutes.ts#L149)

@SanketD92 , @maghirardelli , @kpark277 , @aws-tyler thoughts?

Reply:
Having the check at the service level as well would be great, but I think we can skip for now to minimize how much code we're changing for launch. Only nit is to combine this with the previous line and instead make the error message `Cannot start environment while environment is in ${env.status} state`

Reply:
keeping error checks on the route level for now